pages/login.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+import Page from '../components/Page';
+import { Form, Input, Button, Checkbox } from 'antd';
+import { MailOutlined, LockOutlined } from '@ant-design/icons';
+import axios from '../axios';
+
+const Login: FC = () => {
+
+    const onFinish = values => {
+        const { email, password } = values;
+
+        axios.post('/login', { email, password })
+            .then(() => {
+                console.log('Login Success:', values);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
+
+    return (
+        <Page title="Login">
+            <div className="login-container">
```

Comment:
Why do we have a `login-container` class, we could just ANTD's container component, right?

Reply:
We can. Will make the changes and send a PR again.

Reply:
Unfortunately looks like there is no specific container component in antd. How do I proceed?
.